srcpkgs/motif/template

```diff
@@ -0,0 +1,68 @@
+# Template file for 'motif'
+pkgname=motif
+version=2.3.8
+revision=1
+build_style=gnu-configure
+build_helper=qemu
+configure_args="--disable-jpeg --disable-png --enable-xft --disable-demos"
+hostmakedepends="pkg-config automake libtool flex"
+makedepends="libX11-devel libXft-devel libXt-devel libXpm-devel libXext-devel xbitmaps"
+short_desc="User interface component toolkit"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="LGPL-2.1-only"
+homepage="https://sourceforge.net/projects/motif/"
+distfiles="${SOURCEFORGE_SITE}/motif/Motif%20${version}%20Source%20Code/motif-${version}.tar.gz"
+checksum=859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7
+replaces="lesstif>=0"
+patch_args="-Np1"
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" $makedepends"
+fi
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i lib/Xm/Makefile.am -e 's|$(top_builddir)/config/util/makestr|/usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static &|'
+		vsed -i tools/wml/Makefile.am -e 's|\./wmluiltok|/usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static &|'
+		vsed -i tools/wml/Makefile.am -e 's|\./wml |/usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static &|'
+		vsed -i tools/wml/Makefile.am -e '/motif.wmd:/a	\\tsed -i "s|      exec|& /usr/bin/qemu-${XBPS_TARGET_QEMU_MACHINE}-static |" wmldbcreate'
+	fi
```

Comment:
I suggest moving this block to `post_patch`, so the changes can be verified after `xbps-src patch` wihthout requiring configuration.